src/components/BlogPostFooter.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import STBannerBackdrop from "../images/supertokens-logo-backdrop.png";
+
+const BlogPostFooter = ({ idSlug }) => {
+
+  return (
+    <footer id={`last_section_${idSlug}`}>
+      <b>Written by the Folks at <a href="https://supertokens.com">SuperTokens</a> — hope you enjoyed! We are always available on our Discord server. Join us if you have any questions or need any help.</b>
+      <div className="blog-post-sign-up-banner">
+        <img
+          src={STBannerBackdrop}
+          alt="Blog post sign up banner backdrop"
+          className="sign-up-banner-backdrop"
+        />
+
+        <div className="sign-up-banner-content">
+          <h2 className="sign-up-banner-content-title">Get started for free</h2>
```

Comment:
I prefer this not be an `h2`. If we add a TOC plugin in the future then this will start showing up

Reply:
Right